sql/moz-fx-data-shared-prod/search_terms_derived/search_term_data_validation_reports/init.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE IF NOT EXISTS
+  `moz-fx-data-shared-prod.search_terms_derived.search_term_data_validation_reports`(
+     from_sanitization_job_finished_at timestamp,
+     started_at timestamp,
```

Comment:
is this the job start time? if so, what do you think about making these both `job_finished_at` and `job_started_at`? I think that would clarify the meaning between the two

Reply:
`started_at` is the job start time, but `from_sanitization_job_finished_at` means that this row of data contains validation data pertaining to the sanitization job that finished at that timestamp. It's effectively a foreign key.